dev-docs/bidders/admatic.md

```diff
@@ -14,9 +14,12 @@ gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
 schain_supported: true
 dchain_supported: false
 userIds: all
+prebid_member: true
+pbs_app_supported: true
```

Comment:
Can you link the PR in prebid-server that enabled app support=

Reply:
> Can you link the PR in prebid-server that enabled app support=
PR = https://github.com/prebid/prebid-server/pull/3654
https://github.com/prebid/prebid-server/blob/master/static/bidder-info/admatic.yaml